patches/buildroot/0001-disable-shared-for-host-builds-of-gmp-isl-mpc-mpfr.patch

```diff
@@ -1,7 +1,7 @@
-From ce9da3957e01aa81af49210bb7fec3d8cb440690 Mon Sep 17 00:00:00 2001
+From dc72132b09daa401e9631c9af6463d93f035682d Mon Sep 17 00:00:00 2001
 From: Ben Cressey <bcressey@amazon.com>
 Date: Sun, 6 Aug 2017 23:15:02 +0000
-Subject: [PATCH 1/5] disable shared for host builds of gmp, isl, mpc, mpfr
+Subject: [PATCH] disable shared for host builds of gmp, isl, mpc, mpfr
```

Comment:
Seems odd that `git` numbered the patch but not the subject :sweat_smile:

Reply:
Ben likely passed `--no-numbered` to `git format-patch`. There was a discussion on a PR lately (that I can't find right now). The gist of it is that the patch numbers in the subject line don't add extra information and cause needless changes when adding or removing a patch, so we decided to try without them for a while and see whether we like that better.